sherpa/ui/utils.py

```diff
@@ -1862,12 +1855,7 @@ def set_iter_method(self, meth):
                322
                http://adsabs.harvard.edu/abs/1995ApJ...438..322W
 
-        .. [2] "Bias-Free Parameter Estimation with Few Counts, by
-               Iterative Chi-Squared Minimization", Kearns, Primini, &
-               Alexander, 1995, ADASS IV, 331
-               http://adsabs.harvard.edu/abs/1995ASPC...77..331K
-
-        .. [3] http://iraf.net/irafhelp.php?val=sfit
+        .. [2] http://iraf.net/irafhelp.php?val=sfit
```

Comment:
Line 1844 - unfortunately I can't seem to change it on the GH page - will need to be changed to use `[2]` rather than `[3]` - i.e.

```
        [2]_, where after a fit data points are excluded if the value
```

Reply:
done